ci/jenkins/test.sh

```diff
@@ -55,7 +57,8 @@ Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests o
         --testcase               Windows install OVS, Conformance and Network Policy or Antrea e2e testcases on a Windows or Linux cluster. It can also be flexible ipam or multicast e2e test.
         --registry               The docker registry to use instead of dockerhub.
         --proxyall               Enable proxyAll to test AntreaProxy.
-        --flexible-ipam          Run tests in flexible ipam mode"
+        --flexible-ipam          Run tests in flexible ipam mode
+        --containerd-jumper      Run tests in containerd jumper node on private lab."
```

Comment:
It looks like the ipam, containerd-jump, default cases are mutually exclusive. Maybe we can refactor the command switches to --testbed-type flexible-ipam|jumper|legacy .

Reply:
Updated.